src/components/CustomTableRow/index.tsx

```diff
@@ -54,14 +55,35 @@ const ClipboardWrapper = styled(Clipboard)`
   outline: none !important;
   cursor: pointer;
 `;
+
+const DefaultAttributeNames = [
+  'predecessor',
+  'hash',
+  'block_id',
+  'block_hash',
+  'operation_group_hash',
+  'delegate',
+  'protocol',
+  'context',
+  'operations_hash',
+  'signature'
+];
 interface Props {
   item: any;
   selectedColumns: any[];
   network: string;
+  platform: string;
 }
 
-export const displayType = (network, value, name) => {
-  if (name === 'account_id' || name === 'manager') {
+export const formatValueForDisplay = (platform, network, value, attribute) => {
```

Comment:
Which properties does the attribute parameter contain?

Reply:
https://github.com/Cryptonomic/ConseilJS/blob/feature/governance/src/types/conseil/MetadataTypes.ts#L20

Reply:
can we declare argument types explicitly?